internal/pkg/deploy/cloudformation/stack/transformers.go

```diff
@@ -25,8 +25,13 @@ import (
 )
 
 const (
-	enabled  = "ENABLED"
-	disabled = "DISABLED"
+	enabled           = "ENABLED"
+	disabled          = "DISABLED"
+	standardQueueType = "standard"
+	fifoQueueType     = "fifo"
+	defaultQueueType  = standardQueueType
+	messageGroup      = "messageGroup"
+	perMessageGroupId = "perMessageGroupId"
```

Comment:
nit: what do you think of starting the constants with the service name that it applies to? and separating the `const` blocks:
```suggestion
	efsEnabled           = "ENABLED"
	efsDisabled          = "DISABLED"
	
	sqsStandardQueueType = "standard"
	sqsFIFOQueueType     = "fifo"
	sqsDefaultQueueType  = standardQueueType
	sqsDedupeScopeMessageGroup      = "messageGroup"
	sqsFIFOThroughputLimitPerMessageGroupId = "perMessageGroupId"
```